ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -6,16 +6,28 @@ syntax = "proto3";
 
 package com.daml.ledger.api.v1.admin;
 
+import "com/daml/ledger/api/v1/admin/object_meta.proto";
+import "google/protobuf/field_mask.proto";
+
 option java_outer_classname = "PartyManagementServiceOuterClass";
 option java_package = "com.daml.ledger.api.v1.admin";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
 
-// Status: experimental interface, will change before it is deemed production
-// ready
+// We use 'backing participant' to refer to this specific participant in the methods of this API.
+//
+// Allows inspecting the party management state of the ledger known to the backing participant
+// and managing participant-local party metadata.
+//
+// The authorization rules for its RPCs are specified on the ``<RpcName>Request``
```

Comment:
I like the drive to be more scientific about authorization rules. However, this is a new formalism not used anywhere else in our API. I would not introduce it here. We might want to pick it up in a more rigorous and comprehensive manner later. How about scraping this paragraph and annotating the rpcs (not the requests) instead like so
```
Required authorization: Participant admin
```

Reply:
We use this notation for the user management service already though

Reply:
I have missed that moment. Fair enough then!